Twig/Extension.php

```diff
@@ -118,7 +127,11 @@ private function addDefaultApiCalls()
                 return; // Do not add 'trackPageView'
             }
         }
-
-        $this->paqs[] = ['trackPageView'];
+        if (!empty($lastPaq)) {
+            $this->paqs[] = $lastPaq; // move manually added 'trackPageView' to the end
+            $this->paqs = array_values($this->paqs);
```

Comment:
(Why) do you need this?

Reply:
`array_values()` indexes an array numerically. The keys in `$this->paqs` might not follow a consecutive order at this point, and reindexing the array with `array_values` fixes that. It's more of an eye-candy purpose in this situation.